release_notes/ocp_3_6_release_notes.adoc

```diff
@@ -0,0 +1,2419 @@
+
+We offer new cluster variables per framework area so that you can use different
+time-frames for different framework components. Once set, issue the new
+`redeploy-openshift-ca` playbook.
+
+.New Cluster Variables
+----
+# CA, node and master certificate expiry
+openshift_ca_cert_expire_days=1825
+openshift_node_cert_expire_days=730
+openshift_master_cert_expire_days=730
+
+# Registry certificate expiry
+openshift_hosted_registry_cert_expire_days=730
+
+# Etcd CA, peer, server and client certificate expiry
+etcd_ca_default_days=1825
+
+$ ansible-playbook -i inventory openshift-ansible/playbook/byo/openshift-cluster/redeploy-openshift-ca.yml
+----
+
+[[ocp-36-general-stability]]
+==== General Stability
+
+{product-title} engineering and the OpenShift Online operations teams have been
+working closely together to refactor and enhance the installer. The
+{product-title} 3.6 release includes the culmination of those efforts, including:
+
+- Upgrading from {product-title} 3.5 to 3.6
+- Idempotency refactoring of the configuration role
+- Swap handling during installation
+- All BYO playbooks pull from a normalized group source
+- A final port of operation’s Ansible modules
+- A refactoring of excluder roles
+
+[[ocp-36-metrics-and-logging]]
+=== Metrics and Logging
+
+[[ocp-36-removing-metrics-deployer-and-removing-logging-deployer]]
+==== Removing Metrics Deployer and Removing Logging Deployer
+
+The metrics and logging deployers were replaced with `playbook2image` for `oc
+cluster up` so that `openshift-ansible` is used to install logging and metrics:
+
+----
+$ oc cluster up --logging --metrics
+----
+
+Check metrics and logging pod status:
+
+----
+$ oc get pod -n openshift-infra
+$ oc get pod -n logging
+----
+
+[[ocp-36-expose-elasticsearch-as-a-route]]
+====  Expose Elasticsearch as a Route
+
+By default, Elasticsearch deployed with OpenShift aggregated logging is not
+accessible from outside the logging cluster. This enables a route for external
+access to Elasticsearch for those tools that want to access its data.
+
+Direct access to Elasticsearch using only your OpenShift token. You have the
+ability to provide the external Elasticsearch and Elasticsearch Operations host
+names when creating the server certificate (similar to Kibana). Ansible tasks
+simplify route deployment.
+
+[[ocp-36-developer-experience]]
+=== Developer Experience
+
+[[ocp-36-automicatic-build-pruning]]
+==== Automatic Build Pruning
+
+Previously, only `oc adm prune` could be used. Now, you can define how much
+build history you want to keep per build configuration. Also, you can set
+`successful` versus `failed` separately.
+
+See
+xref:../dev_guide/builds/advanced_build_operations.adoc#build-pruning[Advanced
+Build Operations] for more information.
+
+[[ocp-36-easier-custom-slave-configuration-for-jenkins]]
+==== Easier Custom Slave Configuration for Jenkins
+
+In {product-title} 3.6. making changes to the Jenkins configuration does not
+require new image layers.
```

Comment:
change this to: In openshift 3.6 it's now easier to make images available as slave pod templates.